lib/checkPullRequestBranch.js

```diff
@@ -27,13 +27,43 @@ const checkBranch = async (context) => {
     return branchName.startsWith(prefix);
   });
 
-  if (branchIsInvalid) {
+  if(branchName=='develop')
+  {
     const prAuthor = pullRequest.user.login;
     // Comment on the pull request.
     const wiki = 'wiki'.link(
       'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
       'instructions-for-making-a-code-change'
     );
+    
+    const commentParams = context.repo({
+      issue_number: pullRequest.number,
+      body:
+        'Hi @' + prAuthor +
+        ', PRs made from develop branch are not ' +
+        'allowed. So this PR is being closed. Please make your changes in ' +
+        'another branch and send in the PR. To learn more about contributing ' +
+        'to Oppia, take a look at our ' +
+        wiki + ' (Rule 1 specifically). Thanks!',
+    });
+
+    await context.github.issues.createComment(commentParams);
+    // Close the pull request.
+    const closeIssueParams = context.repo({
+      issue_number: pullRequest.number,
+      state: 'closed',
+    });
+    await context.github.issues.update(closeIssueParams);
+  }
+
+  else if (branchIsInvalid) {
```

Comment:
I believe the message for this section needs to be updated as well since we are now handling develop branch changes separately right?
Also, can we have separate `else if` blocks for each of the scenarios as suggested in the issue, https://github.com/oppia/oppiabot/issues/147?

Reply:
@jameesjohn there are two blocks one for checking develop branch and another for invalid prefix branches. Are you suggesting adding cases for each develop,test and release prefix branches with their separate messages??

Reply:
Yes.
I think that was the aim of the issue, and that will be clearer.

Reply:
Has this been fixed?

Reply:
@riyasingh1004 ping.